contribute/development_environment/README.md

```diff
@@ -116,30 +116,40 @@ brew install jq \
 ```
 
 You can then follow the provided instructions. As you'll see, you need to add
-the following lines to the profile of your shell (eg `~/.bash_profile`):
+the following lines to the profile of your shell (eg `~/.bash_profile` or
+`~/.zprofile`):
+
+**Warning**: for Apple Silicon macs, the new default location for homebrew
+is `/opt/homebrew`, not `/usr/local`. See [discussion](https://github.com/Homebrew/brew/issues/9177).
+Please choose the right definition of `HOMEBREW_PATH` for the following section:
+
+* Intel:  `export HOMEBREW_PATH="/usr/local"`
+* Apple Silicon: `export HOMEBREW_PATH="/opt/homebrew"`
```

Comment:
```
BREW_PREFIX=$(brew --prefix)
```
does the detection.

It also doesn't need to be exported, and probably can be used at the end of the code snipplet.

Reply:
ah, saw this after my own correction.. Thanks